### DISCUSSION TITLE
Reverted to previous commit

### DISCUSSION THREAD

**Discussion by tbarbette** [tbarbette/fastclick#355] [General]
Hi all,
I, unfortunately, pushed to the main branch some work in progress. I forced-push because this history could not be kept. Hence if you pulled in the last days, you should revert to the current state.

Cheers,
Tom

**Comment by hedj17** [comment]
I notice that you have added some new elements in this framework, but you don't update offical document in wiki page.  This brings some inconvenience in use.  Could you update it?

**Reply by tbarbette** [reply]
I updated the wiki for now, I'll add a proper CI when I have time.

**Comment by tbarbette** [comment]
Ah yes ! Thank you for the reminder. I'll create a proper issue.

**Reply by hedj17** [reply]
I see you used FlowClassifier element in conf/middleclick/firewall.click file,  but I don't find it in elements folder. Therefore, I  suspect this element has been updated.  How should I use it?

**Reply by tbarbette** [reply]
Indeed it became CTXManager / CTXDPDKManager with acceleration. I'll check the vagrant VM to verify it's still up-to-date with latest changes.
